scripts/content-utils/card-utils.ts

```diff
@@ -75,22 +74,41 @@ export function cardLogic(
  * Given a generic card this creates a new event card with updated logic content.
  *
  * @param card A card template that contains artistic content
- * @param [left, right] The left and right world actions
+ * @param [left, right] The left and right modifiers and nextEventCardId
  */
 export function eventCardLogic(
```

Comment:
Alright, this simplifies `eventCardLogic()` as well. Made the API much easier to work with to focus only on the important data rather than exact structures.

Noted here that TS gives great support for passing arrays where specific indices are destructured to parameters. Wouldn't have done this in pure JS, where a wrapping object with key:value would have been necessary to guide the correct usage.